src/utils/configs.js

```diff
@@ -22,9 +22,16 @@ let isAdmin = false;
   const el = document.querySelector(`meta[name='env:${x.toLowerCase()}']`);
   configs[x] = el ? el.getAttribute("content") : process.env[x];
 
-  if (x === "BASE_ASSETS_PATH" && configs[x]) {
+  if (x === "BASE_ASSETS_PATH" && configs["BASE_ASSETS_PATH"]) {
     // eslint-disable-next-line no-undef
-    __webpack_public_path__ = configs[x];
+    __webpack_public_path__ = configs["BASE_ASSETS_PATH"];
+
+    // BASE_ASSETS_PATH might be a relative URL like "/" when it is set in
+    // .env or .defaults.env when running locally. We need to convert that
+    // to an absolute URL.
+    if (configs["BASE_ASSETS_PATH"] && !configs["BASE_ASSETS_PATH"].startsWith("http")) {
```

Comment:
We've already verified that configs[BASE_ASSETS_PATH] is truthy, so the first check is redundant

```suggestion
    if (!configs["BASE_ASSETS_PATH"].startsWith("http")) {
```